fe/fe-core/src/main/java/com/starrocks/server/LocalMetastore.java

```diff
@@ -3984,7 +3984,7 @@ public void modifyTableDynamicPartition(Database db, OlapTable table, Map<String
     public void alterTableProperties(Database db, OlapTable table, Map<String, String> properties)
             throws DdlException {
         Map<String, String> logProperties = new HashMap<>(properties);
-        int partitionLiveNumber = -1;
+        int partitionLiveNumber = TableProperty.INVALID;
         if (properties.containsKey(PropertyAnalyzer.PROPERTIES_PARTITION_LIVE_NUMBER)) {
```

Comment:
If the properties have already been copied, why not use logProperties instead?

Reply:
I think both logProperties and properties will work, so I'll change it to logProperties.